controllers/config/config.go

```diff
@@ -1,15 +1,15 @@
 package config
 
-var Config config
+import "github.com/spf13/viper"
+
+var Config = &config{
```

Comment:
On second thought, my last comments on this were kind of confusing.
We have two options for handling the operator config:
1. Completely remove viper, and assign the `StringVar`s in `main.go` directly to these variables
2. Don't use `StringVar` anywhere and call `viper.Get()` once we have set the correct viper options in `main.go`. Just like the the previous implementation in the `FromViper()` method.

In my opinion `2.` is the better approach, as it allows us to keep handling environment vars.